client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -158,7 +158,7 @@ const StockListComponent: FC = () => {
       sortBy,
       onChangeSortBy: updateSortQuery,
     },
-    [sortBy, plugins.stockColumn?.columns.length || 0]
+    [sortBy, plugins.stockColumn?.columns]
```

Comment:
not a big fan of passing non primitives as dependencies, but this works

Reply:
Hm yeah, I think find here because no editable cells..